webapp/channels/src/components/admin_console/system_user_detail/index.ts

```diff
@@ -6,20 +6,36 @@ import {connect} from 'react-redux';
 
 import type {GlobalState} from '@mattermost/types/store';
 
+import {getUserPreferences} from 'mattermost-redux/actions/preferences';
 import {addUserToTeam} from 'mattermost-redux/actions/teams';
 import {updateUserActive, getUser, patchUser, updateUserMfa} from 'mattermost-redux/actions/users';
-import {getConfig} from 'mattermost-redux/selectors/entities/general';
+import {RESOURCE_KEYS} from 'mattermost-redux/constants/permissions_sysconsole';
+import {getConfig, getLicense} from 'mattermost-redux/selectors/entities/general';
 
 import {setNavigationBlocked} from 'actions/admin_actions.jsx';
 import {openModal} from 'actions/views/modals';
+import {getConsoleAccess} from 'selectors/admin_console';
+
+import {it} from 'components/admin_console/admin_definition';
```

Comment:
I couldn't find any other way in webapp to check if a given user has a specified permission. Because it touches the permission code, I didn't want to tweak it much. But yeah, sounds like if. If you look into the functions implementation,. we actually just need the `consoleAccess` variable, but I'm still using this `it` function so that the logic to check for permission stays at the same place.

Reply:
Wouldn't functions like `haveISystemPermission` work here?

Reply:
We don't use it in system console for some reason so I'd stick to whats already being used there.